custom-scripts/numba.sh

```diff
@@ -0,0 +1,3 @@
+#! /bin/bash
+conda install -y cudatoolkit
+conda install -y numba
```

Comment:
we might get this for free with the container we use for pyspark && sparklyR

Reply:
Looking at this: https://docs.anaconda.com/anaconda/packages/py3.5_linux-64, it seems numba does come by default but cudatoolkit does not.

Will update to reflect that.